tests/syntax-tests/create_highlighted_versions.py

```diff
@@ -16,6 +16,16 @@
     "--italic-text=always",
 ]
 
+BINARY_OPTIONS = [
+    "--show-all",
+    "--no-config",
+    "--style=plain",
+    "--color=always",
+    "--theme='1337'",
+    "--italic-text=always",
+]
+
+binary_folders = ["plaintext.txt"]
```

Comment:
To make this more generally useful, what do you think about the following approach:

When reading one of the `source` folders, we check if a `tests/syntax-tests/source/…/bat_options` file exists. If it does, we read the file and expect each line to be an additional command-line option for `bat`.

So for your use case here, you would create `Plaintext/bat_options` with
```
--show-all
```
as the content.

Reply:
Sure, I'll write this up and we can look at it together.